docs/operating-scylla/rest.rst

```diff
@@ -1,8 +1,27 @@
 
 REST
-----
+------
 
-For each JMX operation, attribute get and set, Scylla exposes a matching REST API. You can interact with the REST API using :code:`curl` or using the Swagger UI available at :code:`your-ip:10000/ui`
+Scylla exposes a REST API to retrieve administrative information from a node and execute administrative operations.
+Examples are configuration status and updates, cluster-level information, and more.
 
+:doc:`Nodetool </operating-scylla/nodetool>` CLI tool interacts, using JMX, with a *scylla-jmx* process, which in turn, uses the REST API to interact with the Scylla process.     
 
+You can interact with the REST API directly using :code:`curl` or using the Swagger UI available at :code:`your-ip:10000/ui`
 
+For example, using the Swagger UI with Scylla Docker:
```

Comment:
```suggestion
The following example shows using the Swagger UI with the Scylla Docker image.
```

Reply:
@annastuchlik its hard for me to parse this suggestion